.travis.yml

```diff
@@ -0,0 +1,27 @@
+---
+language: ruby
+sudo: required
+cache: bundler
+rvm:
+- 2.4.5
+services:
+- docker
+addons:
+  postgresql: '9.5'
+env:
+  global:
+  - DATABASE_URL="postgresql://postgres:@localhost:5432/topological_inventory_test?encoding=utf8&pool=5&wait_timeout=5"
```

Comment:
Do we even need to use the `topological_inventory_test` database here? We could drop the `db:create` and then this would be totally generic.

Reply:
There is no built-in database in Travis...only the postgres user with a blank password.  Their docs even say to do a create database yourself.

Reply:
:worried: We seem to be using a database called `travis` without creating it [in manageiq-postgres_ha_admin](https://github.com/ManageIQ/manageiq-postgres_ha_admin/blob/2ce0140a256296fb9e208fab054abdd488c456cb/spec/server_store_spec.rb#L24). [Latest build](https://travis-ci.org/ManageIQ/manageiq-postgres_ha_admin/jobs/445354598#L603) shows those specs ran, but it's entirely possible we're taking advantage of some non-public db in that case ... I can't remember if I ever saw it documented. I think @yrudman added that originally....

Reply:
@yrudman how did you find out that you could use the `travis` database?

Reply:
I do not remember :(

Reply:
Oh weird...I can't find info on that.  Honestly, I'd just prefer to use our own database.